pyproject.toml

```diff
@@ -150,14 +150,15 @@ norecursedirs = [
   "__pycache__",
   "canvas_cli/templates",
   "custom-plugins",
+  "example-plugins",
   "node_modules",
   "static",
   "static-files"
 ]
 python_files = ["*_tests.py", "test_*.py", "tests.py"]
 
 [tool.ruff]
-exclude = ["canvas_generated/"]
+exclude = ["canvas_cli/templates/", "canvas_generated/"]
```

Comment:
why is this change needed? (we want the code in the templates linted, if we disagree with the linter we should add specific ignores as required)